common/djangoapps/student/views.py

```diff
@@ -2257,3 +2273,14 @@ def change_email_settings(request):
         track.views.server_track(request, "change-email-settings", {"receive_emails": "no", "course": course_id}, page='dashboard')
 
     return JsonResponse({"success": True})
+
+
+def _get_xseries_programs(username, user_enrolled_courses): # pylint: disable=invalid-name
+    # make the call to openedx/core/lib/programs/utils and get the programs
+    #response = get_course_programs_for_dashboard(username, user_enrolled_courses)
+    return {
+        user_enrolled_courses[0]: {'total': 2, 'display_name': 'Demo course programs'},
+        user_enrolled_courses[1]: {'total': 12, 'display_name': 'Testing program'},
+        user_enrolled_courses[3]: {'total': 12, 'display_name': 'Testing program'},
+        user_enrolled_courses[4]: {'total': 12, 'display_name': 'Testing program'},
```

Comment:
`total` isn't currently returned in the data structure from `get_course_programs_for_dashboard` - either needs to be added in #10304 or added in this PR once #10304 has been merged.